app/views/content/life-as-a-teacher/what-you-could-teach/_categories.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="row">
+  <section class="category col-space-m">
+    <section class="container category__cards main-section">
+      <nav class="category__nav-cards">
+        <ul>
+          <%= render Categories::CardComponent.new(heading_tag: "h2", card:
+            OpenStruct.new(
+              title: "Computing",
+              description: "Discover how computing is vital, helping children learn digital skills they'll need throughout life.",
+              path: "/life-as-a-teacher/what-you-could-teach/computing",
+              image: "static/images/content/hero-images/teacher2.jpg"
```

Comment:
I think it looks better/more equally weighted without the image. What do you think @jenhadfield-dfe I don't feel like we have enough subject-specific images to put them against each block?

Reply:
I think for this page it would make sense to have an image in the header and just use cards without images for the subjects. I've just committed a change with an image in the header (using the same one that is on the life as a teacher category card)